packages/app/components/Nav/NavBar.tsx

```diff
@@ -98,11 +102,26 @@ const BarsButton = styled.span`
 `;
 
 interface NavBarProps {
-  course: Course;
+  profile: User;
+  courseId: number;
 }
 
-export default function NavBar({ course }: NavBarProps) {
+export default function NavBar({ profile, courseId }: NavBarProps) {
```

Comment:
instead of passing profile through, I think we can just use `useProfile` hook

Reply:
good catch!